src/WingetCreateCore/Common/Extensions.cs

```diff
@@ -53,5 +57,51 @@ public static bool EqualsIC(this string a, string b)
         {
             return string.Equals(a, b, StringComparison.OrdinalIgnoreCase);
         }
+
+        /// <summary>
+        /// Determines if the object is of dictionary type.
+        /// </summary>
+        /// <param name="o">Object to be checked.</param>
+        /// <returns>Boolean value indicating whether the object is a dictionary type.</returns>
+        public static bool IsDictionary(this object o)
+        {
+            return o is IDictionary &&
+                       o.GetType().IsGenericType &&
+                       o.GetType().GetGenericTypeDefinition().IsAssignableFrom(typeof(Dictionary<,>));
+        }
+
+        /// <summary>
+        /// Determines if the type is a List type.
+        /// </summary>
+        /// <param name="type">Type to be evaluated.</param>
+        /// <returns>Boolean value indicating whether the type is a List.</returns>
+        public static bool IsList(this Type type)
```

Comment:
>IsList [](http://example.com/codeflow?start=27&length=6)

Doesn't this tell you if it's IsEnumerable instead?

Reply:
Yes, you're right, I meant to check for IsEnumerable and not list. Changed extension name